src/web/pool.rs

```diff
@@ -40,7 +40,7 @@ impl BeforeMiddleware for Pool {
 }
 
 pub(crate) enum PoolConnection {
-    R2D2(r2d2::PooledConnection<r2d2_postgres::PostgresConnectionManager>),
+    R2D2(Box<r2d2::PooledConnection<r2d2_postgres::PostgresConnectionManager>>),
```

Comment:
Why was this change made?

Reply:
It’s a size-difference thing, that variant is vastly larger than the other

Reply:
The other is only used in test code so I don't think we have to worry about a size difference.